blazor/scheduler/dimensions.md

```diff
@@ -112,4 +112,41 @@ When height and width of the Scheduler are given as percentage, it will make the
         public Nullable<int> RecurrenceID { get; set; }
     }
 }
-```
\ No newline at end of file
+```
+
+## How to set the full height of the view port
+
+When the height of the Scheduler is set in the CSS unit of Viewport Height(vh), it will make the Scheduler occupy the full height within the viewport.
```

Comment:
When the height of the Scheduler is set in the CSS unit of Viewport Height(vh), it will make the Scheduler to occupy the full height within the viewport.